docs/Device implementation How To.md

```diff
@@ -57,13 +57,1016 @@ This section will make modifications to the basic system and show how to add in
+
+From the NMOS Commmisioning Controller located at http://localhost/admin open the Device browser by clicking on the left `Devices` button.  The NMOS Control Mock Node will be present with name NC-01.  Click on the NC-01 link and locate the  IS-12 control WebSocket:
+
+`ws://127.0.0.1:8080/x-nmos/ncp/v1.0/connect urn:x-nmos:control:ncp/v1.0`
+
+This is the WebSocket used to interact with NMOS Control components.
+
+**Install Chrome WebSocket Plugin**
+
+We will install a Chrome WebSocket extension. Other WebSocket clients will work equally well. 
+
+Details on installing Chrome extensions can be found [here](https://support.google.com/chrome_webstore/answer/2664769?hl=en).  Follow the instructions to open the Chrome Web Store and search for WebSocket King Client.  Install this extension. 
+
+After completing the installation of WebSocket King open the extension in a new browser window.  Copy and paste the WebSocket located in the RDS for the NC-01 NMOS Control Mock node into the connections field and click `connect`.  The `Connect` button should turn to `Disconnect` indicating a successful connection the the NC-01 WebSocket.
+
+Next we will verify the ability to read and write to the NMOS Control components running on the mock node.  We will focus on reading and writing to the Stereo Gain Block and related objects provided by the mock node.  Other aspects of control can also be explored by following the examples in the [IS-12 Specification](https://specs.amwa.tv/is-12/) example section. For purposes of this HOW-TO we will focus on working with the Stereo Control and adding code to extend this control then create a new control and interact with this new control.  
+
+**Open a Session and Obtain Information on Control of Interest**
+
+Use the JSON Command to open a new session to the NC-01 WebSocket control. Copy the JSON formatted message into the For more information about the format refer to [AMWA IS-12 NMOS Control Protocol](https://specs.amwa.tv/is-12/ ). Also note that in an actual system most of the manual steps we are performing here would be performed by an NMOS Controller using the IS-12 specification.  For more information about implementing and IS-12 NMOS Controller see the HOW-TO section for Controller implementations.
+
+
+```
+{
+  "protocolVersion": "1.0.0",
+  "messageType": 0,
+  "messages": [
+    {
+      "handle": 1,
+      "arguments": {
+        "heartBeatTime": 5000
+      }
+    }
+  ]
+}
+
+```
+
+**Expected Output from WebSocket King**
+
+```
+{
+  "protocolVersion": "1.0.0",
+  "messageType": 1,
+  "messages": [
+    {
+      "handle": 1,
+      "result": {
+        "status": 0,
+        "value": 3
+      }
+    }
+  ]
+}
+
+```
+
+We now have a WebSocket session `3` open for our WebSocket King client.  Next retrieve the Control elements and locate the Stereo Gain control block.  
+
+Send the following JSON Formatted command to the NC-01 WebSocket Use the Session ID received in the previous command. In our case `3`:
```

Comment:
Since it's the same fields for all the JSON commands I think I will add it at the top and refer the reader to the repo location where the architecture, protocol etc are more fully defined.